Civi/Api4/Generic/DAOGetAction.php

```diff
@@ -85,9 +85,17 @@ class DAOGetAction extends AbstractGetAction {
    */
   protected $having = [];
 
+  /**
+   * @throws \API_Exception
+   * @throws \CRM_Core_Exception
+   */
   public function _run(Result $result) {
     // Early return if table doesn't exist yet due to pending upgrade
     $baoName = $this->getBaoName();
+    if (!$baoName) {
+      // In some cases (eg. site spin-up) the code may attempt to call the api before the entity name is registered.
+      throw new \API_Exception("BAO for {$this->getEntityName()}is not available. This could be a load-order issue");
```

Comment:
Can you add a space after the entity name?

Reply:
Done